pypeit/scripts/show_2dspec.py

```diff
@@ -171,32 +172,32 @@ def main(args):
     # TODO: JFH For some reason Ginga crashes when I try to put cuts in here.
     viewer, ch = ginga.show_image(image, chname=chname_skysub, waveimg=waveimg,
                                   bitmask=mask_in) #, cuts=(cut_min, cut_max),wcs_match=True)
-
-    if sobjs is not None:
-        show_trace(sobjs, args.det, viewer, ch)
+    if not args.clean:
+        if sobjs is not None:
+            show_trace(sobjs, args.det, viewer, ch)
     ginga.show_slits(viewer, ch, tslits_dict['slit_left'], tslits_dict['slit_righ'], slit_ids)
-                    #, args.det)
+
 
     # SKRESIDS
     chname_skyresids = 'sky_resid-det{:s}'.format(sdet)
     image = (sciimg - skymodel) * np.sqrt(ivarmodel) * (mask == 0)  # sky residual map
     viewer, ch = ginga.show_image(image, chname_skyresids, waveimg=waveimg,
                                   cuts=(-5.0, 5.0), bitmask = mask_in) #,wcs_match=True)
-    if sobjs is not None:
-        show_trace(sobjs, args.det, viewer, ch)
+    if not args.clean:
+        if sobjs is not None:
+            show_trace(sobjs, args.det, viewer, ch)
     ginga.show_slits(viewer, ch, tslits_dict['slit_left'], tslits_dict['slit_righ'], slit_ids)
-                    #, args.det)
 
     # RESIDS
     chname_resids = 'resid-det{:s}'.format(sdet)
     # full model residual map
     image = (sciimg - skymodel - objmodel) * np.sqrt(ivarmodel) * (mask == 0)
     viewer, ch = ginga.show_image(image, chname=chname_resids, waveimg=waveimg,
-                                  cuts = (-5.0, 5.0), bitmask = mask_in) #,wcs_match=True)
-    if sobjs is not None:
-        show_trace(sobjs, args.det, viewer, ch)
+                                  cuts = (-5.0, 5.0), bitmask = mask_in)
+    if not args.clean:
+        if sobjs is not None:
```

Comment:
`if not args.clean and sobjs is not None:`

Reply:
fixed